### PR TITLE
Change Sentry log 'environment' tag value.

### PR DESCRIPTION
**Pull request by sumitsum**
## Description
1. Change Sentry log 'environment' tag value to 'Properties' from 'properties'.

Fixes appsmithorg/appsmith #1737 

## Type of change
- Bug fix (non-breaking change which fixes an issue)

## How Has This Been Tested?
Screenshot 2020-11-18 at 9 35 20 PM

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: app/server/appsmith-server/src/main/resources/application.properties [hunk 1]
```diff
@@ -32,6 +32,7 @@ segment.writeKey=${APPSMITH_SEGMENT_KEY:}
 sentry.dsn=${APPSMITH_SENTRY_DSN:}
 sentry.send-default-pii=true
 sentry.debug=off
+sentry.environment=${APPSMITH_SENTRY_ENVIRONMENT:}
```
Comment (mohanarpit) [inline app/server/appsmith-server/src/main/resources/application.properties:35]: Where will the environment `APPSMITH_SENTRY_ENVIRONMENT` be defined? Can you please add it to the file `.env.example` in the project root?

Comment (sumitsum) [reply app/server/appsmith-server/src/main/resources/application.properties:35]: Done.

Comment (mohanarpit) [reply app/server/appsmith-server/src/main/resources/application.properties:35]: Also make the change in `.env.example` in the root of the Appsmith project. Not just the server folder.

Comment (sumitsum) [reply app/server/appsmith-server/src/main/resources/application.properties:35]: Done.

**Review by trishaanand — APPROVED** [review]

**Pull request merged**